Formula/goffice.rb

```diff
@@ -35,6 +35,12 @@ class Goffice < Formula
   uses_from_macos "libxslt"
 
   def install
+    on_linux do
+      # Needed to find intltool (xml::parser)
+      ENV.prepend_path "PERL5LIB", "#{Formula["intltool"].libexec}/lib/perl5"
```

Comment:
```suggestion
      ENV.prepend_path "PERL5LIB", Formula["intltool"].libexec/"lib/perl5"
```
Let's try to be consistent between this line and the next. 😁